### PR TITLE
Simplify provider subclasses of Ovirt

### PR DESCRIPTION
**Pull request by agrare**
There are a number of classes and/or methods which have to be copied completely to work properly due to hard coded references to Ovirt::InfraManager / "ovirt"

Dependents:
* https://github.com/ManageIQ/manageiq-providers-red_hat_virtualization/pull/18

### CODE REVIEW

**Comment by agrare** [comment]
@miq-bot cross-repo-tests https://github.com/ManageIQ/manageiq-providers-red_hat_virtualization/pull/18

**Comment by agrare** [comment]
There is still more to go, but I don't want these PRs to be enormous.

Taking out of WIP

File: app/models/manageiq/providers/ovirt/infra_manager.rb [hunk 1]
```diff
@@ -112,6 +117,14 @@ def self.description
     @description ||= "oVirt".freeze
   end
 
+  def self.vm_vendor
+    "ovirt".freeze
```
Comment (Fryguy) [inline app/models/manageiq/providers/ovirt/infra_manager.rb:121]: Super minor but should these be memoized like the ones above?

**Comment by Fryguy** [comment]
One minor comment, but merging anyway

**Pull request merged**